spec/ddtrace/contrib/rails/support/database.rb

```diff
@@ -7,53 +7,74 @@ module Database
           module_function
 
           def load_adapter!
-            begin
-              require 'pg'
-              connector = 'postgres://postgres:postgres@127.0.0.1:55432/postgres'
+            [
+              'pg',
+              'mysql2',
+              'activerecord-jdbcpostgresql-adapter',
+              'activerecord-jdbcmysql-adapter'
+            ].each do |adapter|
+              begin
+                require adapter
 
-              # old versions of Rails (eg 3.0) require that sort of Monkey Patching,
-              # since using ActiveRecord is tricky (version mismatch etc.)
-              if ::Rails.version < '3.2.22.5'
-                ::Rails::Application::Configuration.class_eval do
-                  def database_configuration
-                    { 'test' => { 'adapter' => 'postgresql',
-                                'encoding' => 'utf8',
-                                'reconnect' => false,
-                                'database' => 'postgres',
-                                'pool' => 5,
-                                'username' => 'postgres',
-                                'password' => 'postgres',
-                                'host' => 'localhost',
-                                'port' => '55432' } }
+                if adapter == 'pg'
+                  connector = postgres_url
```

Comment:
Sorry for not mentioning this yesterday. But this behavior of picking last adapter that successfully loaded seems unnecessary to me - unless I'm missing something.

Do you think we could `return` first adapter that successfully loaded instead ?